src/components/Chat.jsx

```diff
@@ -200,8 +200,6 @@ function Chat() {
     // Clear the messages when the session ends
     setMessages([]);
     sessionStorage.removeItem("chatMessages");
-    sessionStorage.removeItem("sessionId")
-    sessionStorage.removeItem("sessionCode")
```

Comment:
If sessionId and sessionCode are not cleared here, there should be code in end session somewhere that clears them when the session ends. Perhaps when the user clicks on a return to home/start button. However, that may be out of the scope of this PR.